backend/commons/src/test/java/org/sonarsource/sonarlint/core/commons/testutils/GitUtils.java

```diff
@@ -47,7 +48,7 @@ public static Date commit(Git git, String... paths) throws GitAPIException {
     if (paths.length > 0) {
       var add = git.add();
       for (String p : paths) {
-        add.addFilepattern(p);
+        add.addFilepattern(FilenameUtils.separatorsToUnix(p));
```

Comment:
I am not sure we should convert the path here. Windows tests don't make sense if we convert the path here I think